### DISCUSSION TITLE
Merging Samples from Several Treatment Groups

### DISCUSSION THREAD

**Discussion by Melody5528** [stuart-lab/signac#1054] [Q&A] [answered]
Hi,

I need to compare scATAC data among 4 treatment groups, each has 4 samples. I already finished the Cellranger runs for the count data and would like to perform the downstream analysis with Signac.

How do I create the metadata table for the experimental conditions? Are there any coding examples?

Thank you so much for your help!!

**Comment by timoast** [comment, marked as answer]
> How do I create the metadata table for the experimental conditions? Are there any coding examples?

If you want to create a metadata variable describing which cells came from which treatment group, you can add this to each object before merging them. For example:

```r
obj1$treatment <- "a"
obj2$treatment <- "b"
obj <- merge(obj1, obj2)
```

see the merge vignette for more tips about merging chromatin assays: https://satijalab.org/signac/articles/merging.html

**Comment by Melody5528** [comment]
Thanks for your reply. That was very helpful. I appreciated it a lot!
Several people had asked similar questions in the discussions because this issue is not addressed in the vignette. The PBMC example of merging samples did not really exemplify what we are trying to analyze. 
Many of us are beginners who would benefit from a more detailed tutorial with real data for managing multiple samples under several treatment conditions. 
Many thanks again for the wonderful tools!!


    On Friday, April 1, 2022, 12:52:03 PM EDT, Tim Stuart ***@***.***> wrote:  
 
 



How do I create the metadata table for the experimental conditions? Are there any coding examples?


If you want to create a metadata variable describing which cells came from which treatment group, you can add this to each object before merging them. For example:
obj1$treatment <- "a"
obj2$treatment <- "b"
obj <- merge(obj1, obj2)
see the merge vignette for more tips about merging chromatin assays: https://satijalab.org/signac/articles/merging.html
